sdk/storage/Azure.Storage.DataMovement/src/ServiceToServiceJobPart.cs

```diff
@@ -184,6 +184,11 @@ public override async Task ProcessPartToChunkAsync()
             try
             {
                 sourceProperties = await _sourceResource.GetPropertiesAsync(_cancellationToken).ConfigureAwait(false);
+                await _destinationResource.SetPermissionsAsync(
+                    _sourceResource,
+                    sourceProperties,
+                    _cancellationToken).ConfigureAwait(false);
```

Comment:
What about the case where the permissions were set while enumerating? Won't this "re-set" them for every resource in the container on a container transfer?

Reply:
This should be confirmed that it wouldn't do this unless it's been paused then resumed.